### PR TITLE
map extension name to trunk project name

### PR DESCRIPTION
**Pull request by ChuckHend**

### CODE REVIEW

**Review by sjmiller609 — APPROVED** [review]

File: tembo-operator/src/extensions.rs [hunk 1]
```diff
@@ -17,6 +17,30 @@ lazy_static! {
...
+
+// mapping of extension names to their trunk project names
+lazy_static! {
+    pub static ref TRUNK_MAP: HashMap<&'static str, &'static str> = {
+        let mut m = HashMap::new();
+        m.insert("vector", "pgvector");
+        m.insert("embedding", "pg_embedding");
+        m.insert("pgml", "postgresml");
+        m.insert("columnar", "hydra_columnar");
+        m.insert("pg_currency", "pgcurrency");
```
Comment (ianstanton) [inline tembo-operator/src/extensions.rs:29]: I think this one is just `currency`
```suggestion
        m.insert("pg_currency", "currency");
```

Comment (ianstanton) [reply tembo-operator/src/extensions.rs:29]: Sorry, other way around :upside_down_face:

Comment (ChuckHend) [reply tembo-operator/src/extensions.rs:29]: https://pgt.dev/extensions/pg_currency

which extension is this one, @ianstanton ?

Comment (ChuckHend) [reply tembo-operator/src/extensions.rs:29]: typo though, should be `_` right?

Comment (ChuckHend) [reply tembo-operator/src/extensions.rs]: @ianstanton , go ahead and add any other extension you know have a name mismatch in other PRs.

Comment (ianstanton) [reply tembo-operator/src/extensions.rs]: Looks good, that one is `currency`

**Pull request merged**